src/content/blog/2024_zne_catalyst.md

```diff
@@ -0,0 +1,44 @@
+
+Even though Mitiq can be used with PennyLane programs, it doesn't seamless integrate into the Catalyst framework for JIT compilation. In the last few months, Unitary Fund and Xanadu have partnered to improve the integration of error mitigation techniques into the Catalyst compiler. The natural starting point was Zero-Noise Extrapolation (ZNE), which is
+the most popular error mitigation technique in Mitiq, and arguably the simplest to implement, yet very effective.
+
+Romain Moyard and the Xanadu compilation team had already implemented a basic version of ZNE in Catalyst. The Unitary Fund team took it from there, and extended it with additional options.The gist of ZNE is that it estimates the ideal result of a quantum program by 
+- **Stage 1**: Executing the program at increased noise levels
+- **Stage 2**: Extrapolating back to the zero-noise regime.
+
+If you want to know more about the theory behind ZNE, check out [Mitiq documentation](https://mitiq.readthedocs.io/en/stable/guide/zne-5-theory.html).
+
+We worked on the full Catalyst stack and added the following options to the respective stages of ZNE:
+
+- _Local folding_ – Instead of applying noise scaling globally to the entire circuit, noise level increments can now be applied to individual gates, providing finer control over the noise scaling process.
+- _Exponential extrapolation_ – An exponential function is fitted to the execution results of the noise-scaled programs, whereas the previous option only allowed for polynomial extrapolation. Exponential extrapolation can be advantageous in certain noise models where the error compounds exponentially with noise scaling.
+
+Why is this important? It turns out that, depending on the underlying noise models, 
+some folding techniques and some fitting techniques may perform better than others. 
+In complex noise environments, having the flexibility to choose the most effective folding and extrapolation methods can significantly improve error mitigation outcomes. The more complex the noise model is, the more impactful it is to have such a level of customization in ZNE. Should this blog post inspire you to become a better ZNE practitioner, check out [this review paper](https://arxiv.org/abs/2307.05203) for how to make the most out of all the ZNE options.
+
+After implementing these options, it was time to test them. Catalyst programs can already be run on Amazon Braket and error mitigation can be leveraged to attenuate the hardware noise there. Running on real hardware can be time- and money-expensive though. 
+We needed a workflow with better feedback loop time, and for that, we turned into adding a simple depolarizing noise model to Unitary Fund's in-house simulator Qrack. If you missed Dan Strano's [post](https://unitary.fund/posts/2024_qrack_catalyst/) on Qrack integration with Catalyst, go and check it out!
```

Comment:
What do you mean by feedback loop time here?

Reply:
What I mean by 'feedback loop time' is the 'time to smoke test the change end-to-end locally'. I call it feedback loop, because the outcome of the smoke test could be something to fix in the code. I will rephrase it.